runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -234,6 +235,13 @@ BaseLoader<LoaderDomain, SpecificLoader>::BaseLoader::tensorTypeToDataType(const
   }
 }
 
+template <typename LoaderDomain, typename SpecificLoader>
+ir::OperandIndex
+BaseLoader<LoaderDomain, SpecificLoader>::BaseLoader::tensorIdxToOperandIdx(int32_t tensorIdx)
+{
+  return tensorIdx == -1 ? ir::OperandIndex(ir::OPTIONAL_OPERAND) : _tensor_to_operand[tensorIdx];
```

Comment:
Question!
Assuming that an op `Foo` can take params, p1, p2 and p3 where p1 is mandatory and p2 and p3 are optional
and in a model, `Foo(p1, p2)` is used. 
Then I wonder `optional` param of `Foo` means p2 and p3
or only p3 only.
If the code only means p3, a term `optional` for p3 only is quite confusing because p2 is also optional (but user passed p2 param).